src/main/java/com/example/sixthmafiabot/validators/UserValidator.java

```diff
@@ -16,71 +17,73 @@
 import java.util.concurrent.CompletableFuture;
 
 @Component
+@Slf4j
 public class UserValidator extends BaseValidator {
 
     @Autowired
     UserRepository userRepository;
 
     @Async("asyncExecutor")
-    public void validateUser(User user) throws ValidationException{
-        Set<ConstraintViolation<User>> violations =
+    public void validateUser(UserDTO user) throws ValidationException{
+        Set<ConstraintViolation<UserDTO>> violations =
                 validator.validate(user);
 
         if (!violations.isEmpty()) {
             StringBuilder sb = new StringBuilder();
-            for (ConstraintViolation<User> constraintViolation : violations) {
+            for (ConstraintViolation<UserDTO> constraintViolation : violations) {
                 sb.append(constraintViolation.getMessage());
             }
             throw new ValidationException("Error occurred: " + sb.toString());
         }
     }
 
     @Async("asyncExecutor")
-    public void validateIfAlreadyExists(User user) throws AlreadyExistsExcepeion {
+    public void validateIfAlreadyExists(UserDTO user) throws AlreadyExistsExcepeion {
 
         boolean alreadyExists =  userRepository
                 .getUserByTelegramId(user.getTelegramId())
                 .join() != null;
 
         if(alreadyExists){
+
             throw new AlreadyExistsExcepeion("User with identifier = %s already exists"
```

Comment:
make exception value dictionary
e.g. AlreadyExistsExcepeion({"telegramId":"User with this telegramId already exisits"})

Reply:
+